frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import DeckGL from "deck.gl"
+import { JSONConverter } from "@deck.gl/json"
+import * as layers from "@deck.gl/layers"
+import * as aggregationLayers from "@deck.gl/aggregation-layers"
+import Immutable from "immutable"
+import { StaticMap } from "react-map-gl"
+import FullScreenWrapper from "components/shared/FullScreenWrapper"
+import "mapbox-gl/dist/mapbox-gl.css"
```

Comment:
Don't you need to add this scss file?
https://github.com/streamlit/streamlit/commit/11d219b0405953ee9f0f7b00cd6d6b85fdbbebe3#diff-028a1ac45d1cb5284bc51fdc0faf32ad

Reply:
No. is not necessary